int8 channel
float32 angle
int16 speed
---
bool finished
